src/seedu/addressbook/ui/TextUi.java

```diff
@@ -169,4 +164,38 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /**
+     * formatter class to format text for display
+     */
+    static class Formatter{
+
+        /**
```

Comment:
First line of the method header should have a brief description of what the method does